NuRadioMC/simulation/simulation.py

```diff
@@ -1044,6 +1070,13 @@ def _read_input_hdf5(self):
                     self._fin[key] = np.array(value)
         for key, value in iteritems(fin.attrs):
             self._fin_attrs[key] = value
+
+        # copy information over to generator attributes
```

Comment:
it took me a while to understand what you do here. Can you improve the comment? Something like "store all relevant attributes of the input file in a dictionary"

Reply:
of course